docs/pages_en/usage/deploy/deployment_order.md

```diff
@@ -209,13 +209,13 @@ werf converge
 
 In this case, the `first` hook will be deployed first, followed by the `second` hook, and then the `third` hook.
 
-**По умолчанию при повторных развертываниях того же самого хука старый хук в кластере удаляется прямо перед развертыванием нового хука.** Этап удаления старого хука можно изменить аннотацией `helm.sh/hook-delete-policy`, которая принимает следующие значения:
+**By default, redeploying the same hook will delete the old hook in the cluster just before deploying the new one.** You can change the deletion policy for the old hook using the `helm.shook/hook-delete-policy` annotation. It can have the following values:
```

Comment:
```suggestion
**By default, redeploying the same hook will delete the old hook in the cluster just before deploying the new one.** You can change the deletion policy for the old hook using the `helm.sh/hook-delete-policy` annotation. It can have the following values:
```